src/host/nodeconnections.h

```diff
@@ -213,31 +215,33 @@ namespace asynchost
           auto data_to_send = data;
           auto size_to_send = size;
 
-          // If the message is a raft append entries message, affix the
+          // If the message is a consensus append entries message, affix the
           // corresponding ledger entries
           if (
             serialized::read<ccf::NodeMsgType>(data, size) ==
-              ccf::NodeMsgType::consensus_msg &&
-            serialized::peek<raft::RaftMsgType>(data, size) ==
-              raft::raft_append_entries)
+                ccf::NodeMsgType::consensus_msg &&
+              serialized::peek<raft::RaftMsgType>(data, size) ==
+                raft::raft_append_entries ||
+            serialized::peek<pbft::PbftMsgType>(data, size) ==
+              pbft::pbft_append_entries)
```

Comment:
This is prone to breaking in future - `read` advances the data markers while `peek` does not, so we're relying on short-circuit execution in a multi-term expression. I think we should do `msg_type = serialized::read<ccf::NodeMsgType>(data, size); if (msg_type == consensus_msg && (<peek says raft> || <peek says pbft>)) {...}`